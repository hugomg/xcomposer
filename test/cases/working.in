#This is a very small sample file

U03B1	α	aa	# alpha

# Underscore means space
U2192	→	->_

# Backslashes escape
U230A	⌊	[\_
U230B	⌋	\]\_

# Certain keys have a dead and a non-dead version
U2191	↑	|^_