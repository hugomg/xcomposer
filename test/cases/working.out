# THIS FILE WAS AUTOMATICALLY GENERATED BY xcomposer
# DO NOT MODIFY IT BY HAND

include "%L"

<Multi_key> <U0061> <U0061> : "α"
<Multi_key> <U002D> <U003E> <U0020> : "→"
<Multi_key> <U005B> <U005F> : "⌊"
<Multi_key> <U005D> <U005F> : "⌋"
<Multi_key> <U007C> <U005E> <U0020> : "↑"
<Multi_key> <U007C> <dead_circumflex> <U0020> : "↑"
